File: SkyEditor.SaveEditor/MysteryDungeon/Rescue/RBStoredItem.vb
Namespace MysteryDungeon.Rescue
    Public Class RBStoredItem
        Public Sub New()
            ItemID = 0
            Quantity = 0
        End Sub
        Public Sub New(itemID As Integer, quantity As Integer)
            Me.ItemID = itemID
            Me.Quantity = quantity
        End Sub

        Public Property ItemID As Integer

        Public Property Quantity As Integer

        Public Overrides Function ToString() As String
            Return $"{Lists.RBItems(ItemID)} ({Quantity})"
        End Function
    End Class
End Namespace

File: SkyEditor.SaveEditor/MysteryDungeon/Rescue/RBStoredItem.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SkyEditor.SaveEditor.MysteryDungeon.Rescue
{
    public class RBStoredItem
    {
        public RBStoredItem()
        {
        }

        public RBStoredItem(int itemID, int quantity)
        {
            ItemID = itemID;
            Quantity = quantity;
        }

        public int ItemID { get; set; }
        public int Quantity { get; set; }

        public override string ToString()
        {
            return $"{Lists.RBItems[ItemID]} ({Quantity})";
        }
    }
}
